eahub/profiles/management/commands/profilesfromlegacyhub.py

```diff
@@ -74,6 +136,77 @@ def handle(self, *args, **options):
                 "("
                 "SELECT uid, MIN(pid) AS pid "
                 "FROM profile "
+                "WHERE type = 'your_views_and_values' "
+                "GROUP BY uid"
+                ") "
+                "AS your_views_and_values "
+                "USING (uid) "
+                "LEFT JOIN "
+                "("
+                "SELECT "
+                "entity_id, "
+                "MAX(coded_cause_area = 1) AS global_poverty, "
+                "MAX(coded_cause_area = 2) AS animal_welfare_and_rights, "
+                "MAX(coded_cause_area = 3) AS long_term_future, "
+                "MAX(coded_cause_area = 4) AS cause_prioritisation, "
+                "MAX(coded_cause_area = 5) AS meta, "
+                "GROUP_CONCAT("
+                "IF("
+                "coded_cause_area IS NULL, "
+                "field_interest_in_causes_and_com_value, "
+                "NULL"
+                ") "
+                "ORDER BY delta "
+                "SEPARATOR '; '"
+                ") "
+                "AS cause_areas_other "
+                "FROM "
+                "("
+                "SELECT "
+                "entity_id, "
+                "delta, "
+                "field_interest_in_causes_and_com_value, "
+                "CASE field_interest_in_causes_and_com_value "
+                "WHEN 'Earn-to-give' THEN 0 "
+                "WHEN 'Entrepreneurship' THEN 0 "
+                "WHEN 'Global poverty' THEN 1 "
+                "WHEN 'Animal welfare' THEN 2 "
+                "WHEN 'Climate change' THEN 3 "
```

Comment:
Climate change as long term future? If it's not too difficult (and it might well be), I think Climate Change should be its own category.

Reply:
@michaltrzesimiech? @Richenda?